pytensor/graph/op.py

```diff
@@ -289,7 +289,12 @@ def __call__(self, *inputs: Any, **kwargs) -> Variable | list[Variable]:
 
         """
         return_list = kwargs.pop("return_list", False)
+        name = kwargs.pop("name", None)
         node = self.make_node(*inputs, **kwargs)
+        print(name, kwargs)
```

Comment:
Print statements are generally not expected and can be skipped.
@ricardoV94 should we make pre-commit handle this?

Reply:
It seems I missed that before committing changes. Removing this line as it is not required.